tools/dockerbuild/docker/Dockerfile-gctool

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi9/openjdk-21:1.17-2.1705482269
+
+LABEL org.opencontainers.image.source=https://github.com/projectnessie/nessie
+LABEL org.opencontainers.image.description="Projectnessie Nessie GC Tool"
```

Comment:
nit: Maybe `ProjectNessie GC Tool`?

Reply:
I think, lower-case `n` is correct here - don't recall a mention of the word _Projectnessie_ anywhere else - so hard to tell. But there was a two-word projectnessie used by a big online retailer - maybe better to avoid the already unlucky coincidence ;)